### ISSUE TITLE
Mouse controls allow going in reverse

### ISSUE THREAD

**Issue by finnbear** [SoftbearStudios/mk48#126]
![image](https://user-images.githubusercontent.com/20015102/122630032-de0bd300-d075-11eb-9db1-c42ce52e04f5.png)

Missing functionality reported by HMM

**Comment by caibear** [comment]
When you start holding the mouse we should store if you are in the reverse zone or not; Then, even if you drag your mouse outside the zone you will move the same direction. This prevents accidentally reversing when you just want to turn 180 degrees.

**Comment by finnbear** [comment]
Fixed by https://github.com/SoftbearStudios/mk48/commit/5bf9395f731de60e50bd217026076e5ee3d56919
